dali/python/nvidia/dali/plugin/base_iterator.py

```diff
@@ -21,8 +21,8 @@
 
 def _iterator_deprecation_warning():
     warnings.warn("Please set `reader_name` and don't set last_batch_padded and size manually " +
-                  " whenever possible. This may lead, in some situations, to miss some " +
-                  " samples or return duplicated ones. Check the Sharding section of the "
+                  "whenever possible. This may lead, in some situations, to miss some " +
+                  "samples or return duplicated ones. Check the Sharding section of the "
```

Comment:
lead to _something_
```suggestion
                  "whenever possible. This may lead, in some situations, to missing some " +
                  "samples or returning duplicated ones. Check the Sharding section of the "
```

Reply:
Done